docs/how-to-guides/monitoring-with-prometheus-operator.md

```diff
@@ -0,0 +1,139 @@
+# Setting up monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Deploy Prometheus Operator](#deploy-prometheus-operator)
+  * [Configure Prometheus Operator](#configure-prometheus-operator)
+  * [Install Prometheus Operator](#install-prometheus-operator)
+* [Accessing Prometheus Operator sub-components](#accessing-prometheus-operator-sub-components)
+  * [Accessing Prometheus](#accessing-prometheus)
+  * [Accessing Alertmanager](#accessing-alertmanager)
+  * [Accessing Grafana](#accessing-grafana)
+    * [Using port forward](#using-port-forward)
+    * [Using Ingress](#using-ingress)
+
+## Introduction
+
+This guide provides installation steps of monitoring stack using prometheus-operator component and ways to access the sub-components like Prometheus, Alertmanager and Grafana.
+
+## Prerequisites
+
+- A Lokomotive cluster is accessible via `kubectl` deployed on a supported provider.
+
+<!---
+TODO: Once we have tutorials on how to deploy and configure rook or openebs, then point the following to those tutorials.
+-->
+
+- Deploy a storage provider component (rook and rook-ceph or openebs-operator and openebs-storage-class) with default storage class that can cater to the [PVC](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) requests made by prometheus-operator components Alertmanager and Prometheus.
```

Comment:
Again, I'd link to storage components here and maybe provide some sample configuration for them, if we don't have guides to refer to.

Reply:
I think that's why those todos exist. That this is largely work in progress and we will refer to the guides soon enough.

Reply:
Also the work to add rook is largely WIP in https://github.com/kinvolk/lokomotive/pull/494 and can be pointed to from here.